### PR TITLE
Added elCurator documentation

### PR DESCRIPTION
**Pull request by nicosomb**

### CODE REVIEW

File: en/user/import/elCurator.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# From elCurator
+
+## Export your elCurator data
+
+On the settings
+([https://www.elcurator.net/settings>](https://www.elcurator.net/settings))
```
Comment (tcitworld) [inline en/user/import/elCurator.md]: Is the `>` in `settings>` intended ?

File: en/user/import/elCurator.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+page, click on "Préparer mon export" button. A JSON file will be
```
Comment (tcitworld) [inline en/user/import/elCurator.md]: English please!

File: fr/user/import/elCurator.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Depuis elCurator
+
+## Exportez vos données de elCurator
+
+On the settings
```
Comment (tcitworld) [inline fr/user/import/elCurator.md]: No English here please.

File: en/user/import/elCurator.md (hunk 2 above)
Comment (nicosomb) [reply en/user/import/elCurator.md]: not sur that the interface is translated ;-)

Comment (tcitworld) [reply en/user/import/elCurator.md]: Homepage is when requesting English language.

**Comment by j0k3r** [comment]
Looks like the link in the summary doesn't work.
Maybe name the file with an uppercase: `ElCurator.md`

**Pull request merged**